### ISSUE TITLE
[DOCUMENTATION] LinearWorkflow accepts EOTask and (EOTask, name)

### ISSUE THREAD

**Issue by UrielM21** [sentinel-hub/eo-learn#295]
**What is the problem? Please describe.**
Lack of documentation in eolearn.core.eoworkflow.LinearWorkflow

**A clear and concise description of what the problem is:**
Currently, the documentation states that eolearn.core.eoworkflow.LinearWorkflow(*tasks, **kwargs) accepts a variable number of EOTasks

`tasks (EOTask) – Tasks in the order of execution`

However, the code also checks for tuples in the form of (EOTask, name)
In the _init_ method the _parse_tasks method is called which checks for
```
if isinstance(task, EOTask):
    return task, None
if isinstance(task, (tuple, list)) and len(task) == 2:
    return task
```
and you even return an explanatory ValueError:

`ValueError('Cannot parse {}, expected an instance of EOTask or a tuple (EOTask, name)'.format(task))`

**Comment by batic** [comment]
Thank you for reporting. 

Although we try our best, our documentation is often lacking. Would you be willing to extend/correct the docs of this particular EOTask yourself?

**Comment by UrielM21** [comment]
@batic Yes, sure!

**Comment by AleksMat** [comment]
We went ahead and fixed this. Released in version `0.9.2`.
